manifests/default.pp

```diff
@@ -129,3 +129,11 @@
     ensure  => 'running',
     require => File['/etc/init/openfisca.conf'],
 }
+
+if find_file('/opt/mes-aides/vps_use_ssl') or find_file('/opt/mes-aides/use_ssl') {
```

Comment:
This is not satisfying but it is a workaround to bypass an issue encountered on CircleCI (I can't `sudo git clone xx /opt/yy` - cf. https://circleci.com/gh/guillett/test-circle-ci/1 and https://github.com/guillett/test-circle-ci/blob/master/circle.yml)